src/plugin/tests/vite/internalFileWriterPlugin.spec.ts

```
@@ -0,0 +1,733 @@
+  const pathMock = {
+    join: vi.fn((...args) => args.join('/')),
+    dirname: vi.fn((path) => path.split('/').slice(0, -1).join('/')),
+  };
+  return {
+    ...pathMock,
+    default: pathMock,
+  };
+});
+
+// Import after mocking
+import { promises as fs } from 'fs';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)